### PR TITLE
Prepare for v5 release

### PR DESCRIPTION
**Pull request by andreamari**
To be merged after #544 (README update).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,21 +8,38 @@
 % # " - [Bug Fix]"
 % # " - Fix the bug."
 
-## Version 0.5.0 (In Development)
+
+## Version 0.6.0 (February 8th, 2021)
```
Comment (nathanshammah) [inline CHANGELOG.md:12]:
```suggestion
## Version 0.5.0 (February 8th, 2021)
```

**Review by nathanshammah — CHANGES_REQUESTED** [review]
Thanks @andreamari: 
- 0.6--> 0.5 in the changelog
- Need to update the pdf in /docs/pdf

**Review by nathanshammah — APPROVED** [review]

**Comment by nathanshammah** [comment]
@andreamari: the pdf still says v.0.5dev on the first page.

**Comment by andreamari** [comment]
thanks @nathanshammah for the pdf ! :-)

**Pull request merged**